src/DotNetty.Buffers/AbstractByteBuffer.cs

```diff
@@ -302,6 +303,12 @@ public virtual long GetLong(int index)
 
         public virtual char GetChar(int index) => Convert.ToChar(this.GetShort(index));
 
+        public virtual float GetFloat(int index)
+        {
+            int raw = this.GetInt(index);
+            return Unsafe.As<int, float>(ref raw);
```

Comment:
Not sure if unsafe pointer works in not full trusted environment? (UWP?)
Not found another unsafe code in DotNetty.